whisperer.py

```diff
@@ -0,0 +1,31 @@
+def messages(path):
```

Comment:
I would use documentation instead, comments make it look messy
https://swimm.io/blog/code-comments-vs-documentation/

Reply:
I fixed it